framework/util/image_writer.cpp

```diff
@@ -132,13 +135,88 @@ bool WriteBmpImage(
     return success;
 }
 
+bool WriteBmpImageNoAlpha(
+    const std::string& filename, uint32_t width, uint32_t height, uint64_t data_size, const void* data, uint32_t pitch)
+{
+    bool     success   = false;
+    uint32_t row_pitch = width * kImageBpp;
+
+    if (pitch != 0)
+    {
+        row_pitch = pitch;
+    }
+
+    // BMP image data requires row to be a multiple of 4 bytes
+    // Round-up row size to next multiple of 4, if it isn't already
+    const uint32_t rowSizeNoAlpha = ((width * kImageBppNoAlpha) + 3u) & (~0x03);
```

Comment:
To round up to an alignment, please use `util::Platform::GetAlignedSize()`, e.g. `rowSizeNoAlpha = util::Platform::GetAlignedSize(width, kImageBppNoAlpha)`.